test-cases/upgrades/rolling-upgrade.yaml

```diff
@@ -3,12 +3,12 @@
 test_duration: 180
 
 # workloads
-write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM n=60000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..60000000 -log interval=5
-verify_stress_after_cluster_upgrade: cassandra-stress read no-warmup cl=QUORUM n=60000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..60000000 -log interval=5
+write_stress_during_entire_test: cassandra-stress write no-warmup cl=QUORUM n=100000000 -schema 'keyspace=keyspace_entire_test replication(factor=3) compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=1000 -pop seq=1..100000000 -log interval=5
```

Comment:
Why?
What is wrong with 60M that you increase it to 100M?

Reply:
The workload finished after upgraded the second node, we wish it keeps running during the entire upgrade stage. So I tried to increase the pop number, and reduce the rate threads.